demo/scripts/includeSamples.plugin.ts

```diff
@@ -73,7 +75,9 @@ export const includeSamples = (): Plugin => {
 					const frameworkFiles = files[framework];
 					output += `${JSON.stringify(framework)}:{entryPoint:${JSON.stringify(frameworkFiles[0].fileName)},files:{`;
 					frameworkFiles.forEach(({fileName, filePath}) => {
-						output += `[${JSON.stringify(fileName)}]: () => import(${JSON.stringify(filePath + '?raw')}).then(file=>file.default),`;
+						output += `[${JSON.stringify(fileName)}]: () => import(${JSON.stringify(
+							filePath + '?raw'
+						)}).then(file=>file.default).then(file=>file.replace(/@agnos-ui\\/common\\/samples\\/[^/]+/g,'.')),`;
```

Comment:
Unlike the previous code in your PR, this is now also applied when viewing the code in the demo site :+1: , but it is still done at runtime rather than at build time. I think it would be better to do it at built time.
To implement it, you can simply replace `+'?raw'` by something else of your choice (such as `+'?sampleCode'`) and process it in this plugin (in a way similar to [this](https://github.com/vitejs/vite/blob/c7969597caba80cf5d3348cba9f18ad9d14e9295/packages/vite/src/node/plugins/asset.ts#L183)). What do you think?